README.md

```diff
@@ -312,9 +312,10 @@ Nextflow will need access to the working directory where temporary calculations
 ### Transcriptome assembly
 
 ```bash
---assemly                       # switch to transcriptome assemly
+--assembly                      # switch to transcriptome assemlby
```

Comment:
hehe, by fixing the typo you introduced another @fischer-hub  ;) Se the second "assemlby" ;)

Reply:
besides, looks all fine, please feel free to merge! thx!

Reply:
whoops :D